app/invocation/invocation_tabs.tsx

```diff
@@ -26,10 +26,15 @@ export type TabId =
   | "suggestions"
   | "raw"
   | "execution"
-  | "fetches"
-  | "action";
+  | "fetches";
 
 export function getTabId(tab: string): TabId {
+  if (tab === "#action") {
+    // Treat the action tab as the execution tab if executions are enabled,
+    // otherwise treat it as the cache tab.
+    return this.props.executionsEnabled ? "execution" : "cache";
```

Comment:
`this` won't work here since this is a top-level function and not a class method, need to plumb through the `executionsEnabled` prop. For simplicity, I think it's also fine to just return `"execution"` here and have no tabs highlighted when RBE is not enabled.

Reply:
Oh, I did not realize that we need it to be a class method 😮‍💨 

I pushed the fix.